recipe/meta.yaml

```diff
@@ -30,33 +33,56 @@ requirements:
     - {{ cdt('libx11-devel') }}           # [linux]
     - {{ cdt('libxcb') }}                 # [linux]
     - {{ cdt('libxrender-devel') }}       # [linux]
-    - {{ cdt('xorg-x11-proto-devel') }}   # [linux and s390x]
-    - make                                # [unix]
+    - {{ cdt('xorg-x11-proto-devel') }}   # [linux and (s390x or ppc64le or x86_64)]; needed for cairo support
     - meson
```

Comment:
The minimum required version of `meson` is [0.55.3](https://gitlab.gnome.org/GNOME/pango/-/blob/1.50.7/meson.build#L11) and https://gitlab.gnome.org/GNOME/pango/-/blob/1.50.7/NEWS#L202